Trackovid19-web/src/manifest.webmanifest

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "Covidografia",
+    "short_name": "Covidografia",
+    "theme_color": "#ffffff",
```

Comment:
This variable needs to be updated. Probably to the project orange `#fa6400`.

Reply:
This was given by the designer. All the app background is white. So, why you think we should update it?

Reply:
The `theme_color` is not the same as `background_color`.

Reply:
Yes, I know. And again we want it white. 
theme_color - we want that for instance task switcher stays white
background_color - we want that the loading screen white